system-tests/projects/session-and-origin-e2e-specs/cypress/e2e/session/restores_saved_session.cy.js

```diff
@@ -21,3 +21,26 @@ it('t2', () => {
 
   cy.log('after')
 })
+
+// https://github.com/cypress-io/cypress/issues/22381
+describe('test sessionid integrity is maintained', () => {
```

Comment:
Why system tests and not driver or cy-in-cy tests?

Reply:
These are tested as [cy-in-cy tests](https://github.com/cypress-io/cypress/pull/23347/files#diff-4b8a975eae3aec8fa6cff9bdfb404a1c0dac802061c61205f8791166e54c8ac9R186). All of the cy-in-cy tests that I've looked at are loading a project from system-test's to run tests against.

Reply:
Ah my bad. I didn't realize that's how cy-in-cy tests worked. Interesting...

Reply:
No worries at all. I personally don't love the setup...it seems we've started to abuse the system-test setup by copying these projects around and pulling these in to our integration tests. It seems like they should be fixtures instead 🤷🏻‍♀️